ims/common/config.py

```diff
@@ -73,5 +114,6 @@ def section(self, section_name, required=True):
 
 
 class ConfigSection:
+    """ A Simple Object to be used to get cfg.section.option """
     def __init__(self):
```

Comment:
I thought about using object, didn't knew if it was the right way to do it. What if later we decide to add methods to the ConfigSections ? Can't we just add some docs and let it be ?